public/styles/main-styles.css

```diff
@@ -81,6 +85,23 @@ p {
   border-radius: 0;
 }
 
+.header-input-wrapper .input-address::-webkit-input-placeholder {
+  color: #bdbdbd;
+  font-weight: 200;
+}
+.header-input-wrapper .input-address::-moz-placeholder {
+  color: #bdbdbd;
+  font-weight: 200;
+}
+.header-input-wrapper .input-address:-ms-input-placeholder {
+  color: #bdbdbd;
+  font-weight: 200;
+}
+.header-input-wrapper .input-address:-moz-placeholder {
+  color: #bdbdbd;
+  font-weight: 200;
+}
+
 .mdl-layout__header .mdl-textfield__input::placeholder {
```

Comment:
Can you please update the non-vendor-prefixed selector I wrote as well? They should all be consistent I think.